test_cases/test_persons.py

```diff
@@ -1,17 +1,17 @@
 import unittest
-from app.person import Staff, Fellow
+from ..app.person import Staff, Fellow
 
 class TestPerson(unittest.TestCase):
 	"""Test Person added"""
 	def setUp(self):
```

Comment:
They encourage consistency, so it I think this should match the other methods i.e.` set_up()`